modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/exp/IgniteSqlFunctionsTest.java

```diff
@@ -89,17 +89,17 @@ public void testBooleanObjectToBigDecimal() {
     @Test
     public void testPrimitiveToDecimal() {
         assertEquals(
-                new BigDecimal(10),
+                BigDecimal.TEN,
```

Comment:
This doesn't look like a problem, let's go back to how it was.
And others in class.